core/src/com/unciv/models/ruleset/Ruleset.kt

```diff
@@ -328,7 +328,7 @@ class Ruleset {
 
                 val allOtherPrereqs = tech.prerequisites.asSequence().filterNot { it == prereq }.flatMap { getPrereqTree(it) }
                 if (allOtherPrereqs.contains(prereq))
-                    println("No need to add $prereq as a prerequisite of ${tech.name} - it is already implicit from the other prerequisites!")
+                    lines += "No need to add $prereq as a prerequisite of ${tech.name} - it is already implicit from the other prerequisites!"
```

Comment:
This WAS intentional, since I wanted to see if the modders thought this should be a 'breaking' check.
They did, and so this change is correct.

![image](https://user-images.githubusercontent.com/8366208/118399372-87217100-b665-11eb-8389-c6adb964a636.png)